arangosh/Import/ImportHelper.cpp

```diff
@@ -427,17 +429,27 @@ bool ImportHelper::importJson(std::string const& collectionName,
 
       // send all data before last '\n'
       char const* first = _outputBuffer.c_str();
-      char* pos = (char*)memrchr(first, '\n', _outputBuffer.length());
+      char const * pos = (char*)memrchr(first, '\n', _outputBuffer.length());
 
       if (pos != nullptr) {
         size_t len = pos - first + 1;
+        char const * cursor = first;
+        do {
+          ++cursor;
+          cursor = (char*)memchr(cursor, '\n', pos - cursor);
+          ++_rowsRead;
+        } while (nullptr != cursor);
         sendJsonBuffer(first, len, isObject);
         _outputBuffer.erase_front(len);
+        _rowOffset = _rowsRead;
+        ++_numberLines;
```

Comment:
Why do we increase `_numberLines` by just one here, even though we may have read multiple newlines a few lines of code before?
Additionally `_numberLines` is reset as one of the last action of `importJson` anyway, so increasing it by 1 should not have any effect.

Reply:
will remove _numberLines

Reply:
done

Reply:
Also doesn't seem to be done.